metadata/src/utils/validation.rs

```diff
@@ -291,14 +291,8 @@ fn get_extrinsic_hash(
     let address_hash = get_type_hash(registry, extrinsic.address_ty, outer_enum_hashes);
     // The `RuntimeCall` type is intentionally omitted and hashed by the outer enums instead.
     let signature_hash = get_type_hash(registry, extrinsic.signature_ty, outer_enum_hashes);
-    let extra_hash = get_type_hash(registry, extrinsic.extra_ty, outer_enum_hashes);
```

Comment:
No need to validate this because we validate signed exts below. In V16 metadata this type will be removed anyway.